lib/handler/mruby.c

```diff
@@ -45,6 +45,38 @@
 
 #define FREEZE_STRING(v) MRB_SET_FROZEN_FLAG(mrb_obj_ptr(v))
 
+struct st_subreq_ostream_t {
```

Comment:
How about changing the struct name to `st_mruby_subreq_t` or something, assuming that the struct is used to hold all information related to the subrequest?